### ISSUE TITLE
Auth driver does not work

### ISSUE THREAD

**Issue by Samnan** [spadefoot/kohana-orm-leap#33]
When you use leap as the driver (considering everything else properly setup in the application, using Kohana 3.2).

If you use the method Auth::instance()->login( ..... ), the passwords do not match [ Reason: an empty space is contatenated to the user object's password for some strange reason? ]

Furthermore:
If you try to use $user->as_array() in the _login() method of class Base_Auth_Leap, there is another exception:

103             throw new Kohana_InvalidProperty_Exception('Message: Unable to get the specified property. Reason: Property :key is either inaccessible or undefined.', array(':key' => $name));

Just wondering, is the release version tested properly at least once before it's published???

**Comment by bluesnowman** [comment]
The auth module was submitted by another developer this past week and I have been working on improving it.  I will go through it tonight and get it working.

**Comment by CubedEye** [comment]
Hey, sorry about any bugs... I hadn't tested the auth stuff with the new table names, it was working ok with the unchanged column names, I'll have a look into this as well. Let me know if it's been fixed yet?

**Comment by CubedEye** [comment]
Has this been fixed?

As I just took a fresh Kohana install with a fresh copy of LEAP and I used the sql files in it.

I just tried Auth::instance()->login('username', 'password') and Auth::instance()->login('email', 'password') and neither of them threw an error.

I also did a print_r($user->as_array()) inside _login() and after calling Auth::instance()->get_user() in the controller and didn't receive the error stated above.
